lib/unleash/configuration.rb

```diff
@@ -23,7 +23,6 @@ class Configuration
       :bootstrap_config
 
     def initialize(opts = {})
-      ensure_valid_opts(opts)
```

Comment:
I would prefer to keep the `ensure_valid_opts()` method for now.

We can call it from the `validate!` method, to avoid code duplication, if that's a significant concern. Otherwise we can de-duplicate that in a separate PR.

Reply:
This was resolved as discussed at https://github.com/Unleash/unleash-client-ruby/pull/107#discussion_r892358994